File: PYTHON/GATE TWO/Task7.py
print ("Kindly enter 5 numbers")
sum_of_numbers=0

for index in range (1,6,1):

    number=int(input(f"Kindly enter number {index}: "))
    sum_of_numbers +=number
    
print(f"The sum of the five numbers = {sum_of_numbers}") 

File: JAVA/GATE TWO/Task7.java
import java.util.Scanner;

public class Task7{

    public static void main(String...args){
        
        Scanner input= new Scanner(System.in);
        
        System.out.println("Please enter 5 number to sum them: ");
         int sum=0;
        
        for(int index=1; index<=5; index++){
            
            System.out.printf("Kindly enter %d: ", index);
                int number = input.nextInt();
                sum+=number;
        
        }
        
         System.out.printf("The Sum of the Five numbers = %d%n", sum);
 }
 }
